main.py

```diff
@@ -50,12 +49,19 @@
                     metavar='N', help='print frequency (default: 10)')
 parser.add_argument('--resume', default='', type=str, metavar='PATH',
                     help='path to latest checkpoint (default: none)')
-parser.add_argument('--train-size', default=None, type=int, metavar='N',
+parser.add_argument('--train-ratio', default=None, type=float, metavar='N',
                     help='number of training data to be loaded (default none)')
-parser.add_argument('--val-size', default=1000, type=int, metavar='N',
+parser.add_argument('--val-ratio', default=0.1, type=float, metavar='N',
```

Comment:
Can you try to capture the error when `--val-size` and `--val-ratio` are both used?

Reply:
Also, please change the document in README for the new interfaces.

Reply:
Currently the behavior is that when both are used, the --val-size will override. I'll add an error capture if you think necessary.

I'll modify README, but whether to emit an error or to let one override when both options are present must be decided first.

Reply:
Let's explicitly capture this error when both are used. This will avoid confusions to the user.